app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -59,28 +59,46 @@ public String returnAllData() {
     //Getters and Setters
 
     //Pre Match
-    public String getTeamNumber() {    return teamNumber;    }
-    public void setTeamNumber(String teamNumber) { this.teamNumber = teamNumber;    }
-    public String getMatchNumber() {    return matchNumber;    }
-    public void setMatchNumber(String matchNumber) { this.matchNumber = matchNumber;    }
+    public String getTeamNumber() {    
+        return teamNumber;    
+    
+    }
+    public void setTeamNumber(String teamNumber) { 
+        this.teamNumber = teamNumber;   
+    }
+    
+    public String getMatchNumber() {    
+        return matchNumber;    
+    }
```

Comment:
Why is the match number a string?

Reply:
because we receive it as a string and have no reason to turn it into a number since it's only used for data output, no numerical operations